### DISCUSSION TITLE
Error when installing wordpress

### DISCUSSION THREAD

**Discussion by infinityonlinesolutions** [evertramos/server-automation#10] [Q&A]
Hello evertramos,

Greetings, thank you for this significant server solution and packaging using docker. We followed the video https://www.youtube.com/watch?v=lABtNXuHYdc and easy-server to setup server-automation on ubuntu, we have setup the .env files like its recommended in the video, however when we install wordpress we get the following error:

Proxy .env file not found at '/home/root/server/proxy/compose/.env'
 ✘ ERROR   It seems something went wrong running 'new-site.sh'.
   We will try to UNDO all actions done by this script.
   Please make sure everything was back in place as when you started.
 𝝯 WARNING [undo] Creating site folder '/home/root/server/sites/star.ios-templates.com'.
 ✔ success Function 'system_safe_delete_folder'
 
We have tried with a sudo user and root user, and have the same problem in both users.

Any help or guidance on the matter would be great.

Thanks and Best Regards,
Santosh

**Comment by infinityonlinesolutions** [comment]
Hello Again, After some tweaking around, I got the easy-server and the server-automation to work, and to create a wordpress website on docker.

Thank you @evertramos for this incredible work, and for sharing the same.

Thanks and Best Regards,
Santosh

**Comment by evertramos** [comment]
Thanks for your feedback!

**Comment by mshazy** [comment]
thanks
